### PR TITLE
[Layer5ng] calendar inclusion

### PR DESCRIPTION
**Pull request by vineethvanga18**
**Description**
Added calendar 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Build failing... hmm.

**Comment by leecalcote** [comment]
@Tanuj22 which buttons are the standard components that should be reused here? The onhover style on these two buttons doesn't conform to our theme.

File: src/sections/Calendar/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const Calendar = () => {
+
+    return (
+        <CalendarStyleWrapper>
+            <div>
+                <h1>Community Calendar</h1>
+                <p>Join any or all of the weekly meetings</p>
+                <div className="calendar-wrapper">
+                    <div className="calendar-grid">
+                        <FullCalendar
```
Comment (leecalcote) [inline src/sections/Calendar/index.js:17]: How can the button from this FullCalendar component be overridden with the Layer5NG reusecore/button?

Comment (leecalcote) [reply src/sections/Calendar/index.js:17]: @vineethvanga18, will you look into https://fullcalendar.io/docs/css-customization and see that the Layer5ng button style is applied (ideally, not copying / repeating the core button's style)?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by vineethvanga18** [comment]
@leecalcote the checks are now passing, used react-loadable package for client-side imports
// or are we already using any package for client-side loads

**Review by Jashpatel1 — COMMENTED** [review]
@vineethvanga18 Before moving forward please correct the responsiveness.

**Comment by vineethvanga18** [comment]
@Jashpatel1 mobile view for calendar isn't there yet.

File: src/sections/Calendar/index.js (hunk 1 above)
Comment (vineethvanga18) [reply src/sections/Calendar/index.js:17]: @leecalcote is this addressed in your commits?

**Comment by vineethvanga18** [comment]
@leecalcote also when you first load '/community/calendar' you see a "Loading...", a minimal suspense. Should this be changed or let it remain.
Screenshot from 2020-12-30 13-03-48

File: src/sections/Calendar/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import dayGridPlugin from "@fullcalendar/daygrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import googleCalendarPlugin from "@fullcalendar/google-calendar";
+
+const Calendar = () => {
+
+    return (
+        <CalendarStyleWrapper>
+            <div>
+                <h1>Community Calendar</h1>
```
Comment (Tanuj22) [inline src/sections/Calendar/index.js:13]: Use `PageHeader` Component

File: src/sections/Calendar/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                <p className="sub-heading">Join any or all of the weekly meetings</p>
+                <div className="calendar-wrapper">
```
Comment (Tanuj22) [inline src/sections/Calendar/index.js:15]: Use the `Container` Component to wrap this.

**Review by Tanuj22 — COMMENTED** [review]
For the mobile view, just fix the position of the buttons and add scrolling in the x and y direction for the calendar. This should be enough to go-live

File: src/sections/Calendar/index.js (hunk 1 above)
Comment (Tanuj22) [reply src/sections/Calendar/index.js:17]: It's not. Ideally the button in `resusecore/Button` should be used directly instead of just copying the css

**Comment by leecalcote** [comment]
My CSS coloring of the Today button was poorly done. Could use some fixing.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**